src/System.ComponentModel.EventBasedAsync/src/System/ComponentModel/AsyncOperation.cs

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading;
+
+namespace System.ComponentModel
+{
+    public sealed class AsyncOperation
+    {
+        private SynchronizationContext syncContext;
```

Comment:
These fields (and those on some of the other types in this PR) aren't following the naming guidance.